samples/OcelotServiceFabric/src/OcelotApplicationApiGateway/OcelotApplicationApiGateway.csproj

```diff
@@ -1,23 +1,23 @@
-﻿<Project Sdk="Microsoft.NET.Sdk.Web">
-  <PropertyGroup>
-    <Description>Stateless Web Service for Stateful OcelotApplicationApiGateway App</Description>
-    <TargetFramework>net7.0</TargetFramework>
-    <ImplicitUsings>disable</ImplicitUsings>
-    <Nullable>disable</Nullable>
-    <AssemblyName>OcelotApplicationApiGateway</AssemblyName>
-    <PackageId>OcelotApplicationApiGateway</PackageId>
-    <PlatformTarget>x64</PlatformTarget>
-  </PropertyGroup>
-    <ItemGroup>
-    <None Update="ocelot.json;appsettings.json;">
-      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
-    </None>
-  </ItemGroup>
-  <ItemGroup>
-    <PackageReference Include="Microsoft.ServiceFabric" Version="9.1.1799" />
-    <PackageReference Include="Microsoft.ServiceFabric.Services" Version="6.1.1799" />
-  </ItemGroup>
-  <ItemGroup>
-    <ProjectReference Include="..\..\..\..\src\Ocelot\Ocelot.csproj" />
-  </ItemGroup>
+﻿<Project Sdk="Microsoft.NET.Sdk.Web">
+  <PropertyGroup>
+    <Description>Stateless Web Service for Stateful OcelotApplicationApiGateway App</Description>
+    <TargetFrameworks>net6.0;net7.0;net8.0</TargetFrameworks>
+    <ImplicitUsings>disable</ImplicitUsings>
+    <Nullable>disable</Nullable>
+    <AssemblyName>OcelotApplicationApiGateway</AssemblyName>
+    <PackageId>OcelotApplicationApiGateway</PackageId>
+    <PlatformTarget>x64</PlatformTarget>
+  </PropertyGroup>
+  <ItemGroup>
+    <None Update="ocelot.json;appsettings.json;">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </None>
+  </ItemGroup>
+  <ItemGroup>
+    <PackageReference Include="Microsoft.ServiceFabric" Version="10.1.1541" />
+    <PackageReference Include="Microsoft.ServiceFabric.Services" Version="7.1.1541" />
```

Comment:
Everything fine locally during the build?

Reply:
yes